### PR TITLE
185 sql server

### LINKED ISSUE #185: Explore data versioning
need to store historical data, so we can track edits made over time.

### PR DESCRIPTION
**Pull request by jlhogan**
closes #185  by replacing postgres backend with sqlserver backend and leveraging their [temporal tables](https://docs.microsoft.com/en-us/sql/relational-databases/tables/temporal-tables?view=sql-server-ver15)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "jsonwebtoken": "^8.5.1",
     "jwks-rsa": "^1.8.1",
     "moment": "^2.27.0",
+    "mssql": "^6.2.1",
     "multer": "^1.4.2",
     "nodemon": "^2.0.4",
     "pg": "^7.3.0",
```
Comment (JordanGuinn) [inline package.json:29]: We can probably get rid of this `pg` dep as well, presuming SQL Server is good and functional.

File: src/data/initialize.ts [hunk 2]
```diff
@@ -19,69 +19,56 @@ import moment from 'moment';
 export const initialize = async () => {
   const qb = getManager().createQueryBuilder();
 
-  let user;
-  try {
-    const user = getManager().create(User, {
+  let user = await getManager().findOne(User, 1);
```
Comment (JordanGuinn) [inline src/data/initialize.ts:22]: 🎉🎉🎉

**Review by JordanGuinn — APPROVED** [review]
LGTM

File: client/src/containers/EditRecord/Forms/EnrollmentFunding/Form/ChangeEnrollmentForm.tsx [hunk 3]
```diff
@@ -133,7 +133,6 @@ export const ChangeEnrollmentForm: React.FC<ChangeEnrollmentFormProps> = ({
           {!!currentEnrollment && (
             <>
               <h3>Previous enrollment</h3>
-              <h4 className="font-heading-md margin-bottom-0">End date</h4>
```
[the remark this answers is not in the document]
Comment (jlhogan) [reply client/src/containers/EditRecord/Forms/EnrollmentFunding/Form/ChangeEnrollmentForm.tsx:136]: oh i just was looking and saw that this is a duplicated header, but it has nothing to do with this change and i should probably not do it here -- i bet melanie's PR deals with this

**Pull request merged**